initrd/bin/kexec-select-boot

```diff
@@ -1,8 +1,82 @@
 #!/bin/sh
 # Generic configurable boot script via kexec
-set -e -o pipefail
-. /tmp/config
-. /etc/functions
+
+usage() {
+	cat <<End-of-message
+Usage
+=====
+
+Generic configurable boot script via kexec
+	
+$0 -b /boot [options]
+	
+Options
+-------
+
+### -a
+
+Anything provided to the -a argument is appended to the kernel command line (usually comes from parsing the grub, syslinux, BLS files). Ex.
+
+	-a 'export CONFIG_BOOT_KERNEL_ADD="intel_iommu=on intel_iommu=igfx_off"'
```

Comment:
Same as remove comments!
I restate that this, showing export statement itself instead of its board origin, is more misleading then clarifying!